lib/resources/host.rb

```diff
@@ -156,7 +160,8 @@ def ping(hostname, port, protocol)
 
       {
         success: resp.exit_status.to_i.zero?,
-        output: resp.stderr,
+        connection_output: resp.stderr,
```

Comment:
This came up in an earlier comment. It's to hide the implementation details from the end user.